File: starting_filesets/C#/Recently Used List/RecentlyUsedListTest.cs

using NUnit.Framework;

[TestFixture]
public class RecentlyUsedListTest
{
    [Test]
    public void Example()
    {
        int expected = 6 * 9;
        int actual = new RecentlyUsedList().Answer;
        Assert.AreEqual(expected, actual);
    }
}

File: starting_filesets/Java/Recently Used List/RecentlyUsedListTest.java

import org.junit.*;
import static org.junit.Assert.*;

public class RecentlyUsedListTest 
{
    @Test
    public void example()
    {
        int expected = 6 * 9;
        int actual = new RecentlyUsedList().answer();
        assertEquals(expected, actual);
    }
}
